app/views/components/_published-dates.html.erb

```diff
@@ -4,10 +4,13 @@
   history = Array(history)
   last_updated ||= false
   link_to_history ||= false
-  history_class = "app-c-published-dates--history" if history.any?
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
```

Comment:
This is interesting - I'd never imagined the shared helper being used outside of the components gem. My initial reaction was that this isn't a good idea, but actually I can't think of a reason why. Might be interesting to have a chat about this.

Reply:
We talked about it on Slack but just to briefly document what was said: 
Your concern was based on isolation - the applications usually import the components (like I'm using the shared helper) but they don't do it in a way that depends upon their internal structure.

I mentioned there is a precedent to the `shared_helper` being used outside the components gem in other frontend apps such as [finder-frontend](https://github.com/alphagov/finder-frontend/blob/919a38385d5ce526bb4b33449e6cf591ce55db38/app/views/components/_expander.html.erb#L7) and [collections](https://github.com/alphagov/collections/blob/0ef7d799302dfdd6a603f2ddcef130b9f10e2398/app/views/components/_signup-link.html.erb#L10).


We agreed it seems like something that should be raised for discussion at the GOVUK frontend community meeting.